src/nvim/eval.c

```diff
@@ -1147,7 +1147,7 @@ call_vim_function (
       len = 0;
     else
       /* Recognize a number argument, the others must be strings. */
-      vim_str2nr(argv[i], NULL, &len, TRUE, TRUE, &n, NULL);
```

Comment:
`//`-style comment and braces around both `if` and `else`. Since you are changing some line linter will require you to fix lint errors on adjacent lines.